apple/Sources/FerrostarMapLibreUI/Models/NavigationMapViewContentInsetMode.swift

```diff
@@ -12,7 +12,7 @@ public enum NavigationMapViewContentInsetMode {
     /// where the user location should appear toward the bottom of the map.
     ///
     /// This is used to accommodate a top InstructionView
-    case portrait(within: GeometryProxy, verticalPct: CGFloat = 0.75)
+    case portrait(within: GeometryProxy, verticalPct: CGFloat = 0.75, minVertical: CGFloat = 210)
```

Comment:
Aside from the larger discussion on this topic. If we keep this as is, I think we should:

1. Add a param explanation for `minVertical` versus the percentage. Explaining why you'd adjust this value and how the percentage interacts with it. E.g. at the call site, it could be easy to assume percentage is within the height minus minVertical or vise versa.
2. Consider if we want a default value here. It seems likely we've got variable inputs here with the size of the arrival and instruction view?

Just want to make sure this is easy to reason with for someone implementing a slightly custom UI, etc.